examples/zkapps/01-hello-world/src/main.ts

```diff
@@ -1,17 +1,11 @@
 import { Square } from './Square.js';
 import {
```

Comment:
@halsaphi  @LucidSamuel thanks for the updates.
As discussed in comments, let's do all of the required updates here

This PR is still incomplete. All code changes that are reflected in tutorial docs need testing and updating
https://github.com/o1-labs/docs2/blob/main/docs/zkapps/tutorials/01-hello-world.mdx#L305 
and 
https://github.com/o1-labs/docs2/blob/main/docs/zkapps/tutorials/01-hello-world.mdx#L316 
and 
https://github.com/o1-labs/docs2/blob/main/docs/zkapps/tutorials/01-hello-world.mdx#L330
and so on

Please be sure to run the tutorial after code updates to ensure all is as expected. 

Thank you!

Reply:
Thanks for cross-checking @barriebyron, 
I ran the code with my previous PR so isReady wasn't included. 
@qwadratic can you please just update this and also add it to the PR so we could just test and review at once. Thanks :)

Reply:
@LucidSamuel @barriebyron done